roles/nuage_master/tasks/certificates.yml

```diff
@@ -10,7 +10,7 @@
 
 - name: Create the req file
   command: >
-    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ ansible_nodename }}"
+    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ openshift_master_cluster_hostname }}"
```

Comment:
You may want to pass in `openshift.common.cluster_hostname` from the playbooks as a nuage specific variable incase `openshift_master_cluster_hostname` is unset in the inventory.